pubspec.yaml

```diff
@@ -13,3 +13,5 @@ environment:
 dev_dependencies:
   lints: ^1.0.0
   test: ^1.16.0
+dependencies: 
+  posix: ^3.0.0
```

Comment:
Adding dependencies to a core package like `path` comes with a high cost to the ecosystem. We don't plan on adding any non-dev dependencies to `path` at this time.